participants/jacob.json

```
@@ -0,0 +1,44 @@
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Jacob",
+		"familyName": "Ayokunle",
+		"placeFamilyNameFirst": false,
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "aderopoa",
+	// company name (optional)
+	"company": "insure365",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": true,
+		// June 28th, 2025
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": false,
+	// your current interests (JS and in general) (required)
+	"tags": ["Next.js", "React", "Node.js", "Typescript", "Docker", "Kubernetes", "CI/CD", "DevOps", "Startups"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "I build full stack applications with JavaScript. I love to learn new things and share my knowledge with others.",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "Deploying your application with coolify",
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	// the following sizes are available: S, M, L, XL, 2XL, 3XL (only regular cut)
+	"tShirtSize": "L",
+	"linkedin": "https://www.linkedin.com/in/jacobayokunle",
```

I get a 404 with this URL. From the picture on the website, it looks like this might be your actual profile URL?

```suggestion
	"linkedin": "https://www.linkedin.com/in/jacob-ayokunle",
```